utils.py

```diff
@@ -156,11 +156,11 @@ def _mat_mult(X_M, Y_M):
         """
         X_M_row_length=len(X_M[0])
         for row in X_M:
-            assert len(row)==X_M_row_length,"Lenght of the rows of matrix X_M is not uniform"
+            assert len(row)==X_M_row_length,"Length of the rows of matrix X_M is not uniform"
```

Comment:
To conform with the general style of the codebase, add spaces around '==' + after the commas. There are a couple more occasions where this is needed, if you could fix them it would be great.

Reply:
Okay Sir. Working on it.

Reply:
@MrDupin Thank You for your feedback. I have updated in most of my PR submissions. Can you please review them.